scripts/sof-testbench-helper.sh

```diff
@@ -177,16 +184,17 @@ else
     fi
 fi
 
-if [ -n "$OUTWAV" ]; then
+# Generate the outwav if we are keeping our data files for inspection.
+if [[ "$KEEP_TMP" == false ]]; then
+    echo Deleting temporary files
+    rm -f "$INFILE1" "$OUTFILE1" "$TRACEFILE" "$PROFILEOUT"
+else
     echo Converting raw output to "$OUTWAV"
     if [[ "$BITS" == "24" ]]; then
-	sox --encoding signed-integer -L -r "$RATE_OUT" -c "$CHANNELS_OUT" -b 32 "$OUTFILE1" "$OUTWAV" vol 256
+	    sox --encoding signed-integer -L -r "$RATE_OUT" -c "$CHANNELS_OUT" -b 32 \
```

Comment:
The original indentation seems to have been correct. Also this and the following line shouldn't have equal indentation

Reply:
This file is a mixture of tabs/spaces for indentation. Fixed this part.